src/logics/stargate/logic.swap-token.ts

```diff
@@ -10,25 +10,36 @@ import * as common from '@protocolink/common';
 import * as core from '@protocolink/core';
 import {
   getContractAddress,
+  getDstChainIds,
+  getDstTokens,
   getMarkets,
   getPoolDecimals,
   getPoolIds,
+  getSTGToken,
   getStargateChainId,
   isSTGToken,
   supportedChainIds,
 } from './configs';
 import { getNativeToken } from '@protocolink/common';
 
-export type SwapTokenLogicTokenList = Record<string, common.Token[]>;
+interface DstTokenList {
+  chainId: number;
+  tokens: common.Token[];
+}
+
+interface StargateTokenList {
+  srcToken: common.Token;
+  dstTokenLists: DstTokenList[];
+}
+
+export type SwapTokenLogicTokenList = StargateTokenList[];
```

Comment:
Can we simplify the code by embedding the two additional interfaces directly into SwapTokenLogicTokenList for clarity? Such as 

```
export type SwapTokenLogicTokenList = {
  srcToken: common.Token;
  dstTokenLists: {
    chainId: number;
    tokens: common.Token[];
  }[];
}[];
```

Reply:
We followed the Stargate naming convention (eg. dstGasForCall), but we can change to destXXX in our logic.